supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -182,9 +182,9 @@ Admonitions should draw the reader’s attention to certain information. Keep ad
 
 Valid admonition types:
 
-NOTE:: Additional information of which users should be aware, but ignoring it does not impact product usage.
+NOTE:: Additional information, such as product performance, configuration, or support. Ignoring this information does not impact product usage.
```

Comment:
I've read this several times, and it looks like an incomplete thought. What is the information telling you about product performance, configuration, or support?
Trying to blend some ideas from IBM Style and this info...how about:
Additional information that provides guidance or advice that could improve product configuration, performance, or supportability. & etc.